app/models/embed/was_seed_thumbs.rb

```diff
@@ -0,0 +1,46 @@
+module Embed
+  class WasSeedThumbs
+    attr_reader :druid
+    def initialize(druid)
+      @druid = druid
+    end
+
+    def get_thumbs_list
```

Comment:
Do you mind adding comments in yardoc notation? I know we previously did not do this on the other classes, but I think it might be helpful longterm.